### DISCUSSION TITLE
Scrollviewer: Setting Borderbrush and Borderthickness has no effect

### DISCUSSION THREAD

**Discussion by FabianGolembiewski-sp** [AvaloniaUI/Avalonia#12302] [Q&A]
In the ControlTemplate of the Scrollviewer there are missing TemplateBindings for Borderbrush and Borderthickness. Therefore setting the properties has no effect to the Scrollviewer. Found in AvaloniaUI Version 0.10.0-preview5, using Avalonia.Themes.Default/DefaultTheme.

**Comment by timunie** [comment]
Where should this be set? I think this is by design. If you need to have a border visible, I guess one has to override the controltemplate with the needed ControlTemplate. Please find the original one here: 
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Simple/Controls/ScrollViewer.xaml 

Note: as this is imo by design, I'll convert this into a Discussion now.

**Comment by FabianGolembiewski-sp** [comment]
The ControlTemplate of the ScrollViewer.xaml should set Borderbrush and Borderthickness,
as done in other controls:

This would look like:
```
...
<ScrollContentPresenter Name="PART_ContentPresenter"
                                  Padding="{TemplateBinding Padding}"
BorderThickness={TemplateBinding BorderThickness}
BorderBrush={TemplateBinding BorderBrush}
                                  HorizontalSnapPointsType="{TemplateBinding HorizontalSnapPointsType}"
                                  VerticalSnapPointsType="{TemplateBinding VerticalSnapPointsType}"
                                  HorizontalSnapPointsAlignment="{TemplateBinding HorizontalSnapPointsAlignment}"
                                  VerticalSnapPointsAlignment="{TemplateBinding VerticalSnapPointsAlignment}"
                                  Background="{TemplateBinding Background}"
                                  ScrollViewer.IsScrollInertiaEnabled="{TemplateBinding IsScrollInertiaEnabled}">
...
```
Just a reminder...

**Reply by timunie** [reply]
If you think that this should be there out of the box, a PR would be welcome I think.

**Comment by nadirvishun** [comment]
It's still not working in the latest version 11.3.0

**Reply by maxkatz6** [reply]
Feel free to open a PR with proposed changes from above
